doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1455 @@
+==============================
+
+* Added:
+
+  * The ``SB_CONFIG_MCUBOOT_USE_ALL_AVAILABLE_RAM`` sysbuild Kconfig option to system that allows utilizing all available RAM when using TF-M on an nRF5340 device.
+
+    .. note::
+       This has security implications and might allow secrets to be leaked to the non-secure application in RAM.
+
+  * The ``SB_CONFIG_MCUBOOT_NRF53_MULTI_IMAGE_UPDATE`` sysbuild Kconfig option that enables updating the network core on the nRF5340 SoC from external flash.
+  * The AP-Protect sysbuild Kconfig options to enable the corresponding AP-Protect Kconfig options for all images in the build:
+
+    * ``SB_CONFIG_APPROTECT_LOCK`` for the :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK` Kconfig option.
+    * ``SB_CONFIG_APPROTECT_USER_HANDLING`` for the :kconfig:option:`CONFIG_NRF_APPROTECT_USER_HANDLING` Kconfig option.
+    * ``SB_CONFIG_APPROTECT_USE_UICR`` for the :kconfig:option:`CONFIG_NRF_APPROTECT_USE_UICR` Kconfig option.
+    * ``SB_CONFIG_SECURE_APPROTECT_LOCK`` for the :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_LOCK` Kconfig option.
+    * ``SB_CONFIG_SECURE_APPROTECT_USER_HANDLING`` for the :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_USER_HANDLING` Kconfig option.
+    * ``SB_CONFIG_SECURE_APPROTECT_USE_UICR`` for the :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_USE_UICR` Kconfig option.
+
+  * CMake warning when static partition manager file has been changed but changes will not be read until a pristine build is performed.
+  * Encrypted firmware update to :file:`dfu_application.zip` instead of the unencrypted firmware update when encrypted image support is enabled in sysbuild.
+  * Sysbuild-assigned MCUboot image IDs feature, which deals with MCUboot image IDs for different components in a project.
+  * The ``SB_CONFIG_LWM2M_CARRIER_DIVIDED_DFU`` sysbuild Kconfig option that enables the generation of proprietary application update files required for the LwM2M carrier divided FOTA procedure.
+
+* Fixed:
+
+  * An issue with the |NCS| boot banner using ``CMAKE_SOURCE_DIR`` instead of ``APPLICATION_SOURCE_DIR``.
+  * An issue with :kconfig:option:`CONFIG_MBEDTLS_CFG_FILE` having a prompt which would, in some circumstances, lead to a sticky value if the configuration was changed.
+  * An issue with changes to image configurations in a ``sysbuild.cmake`` file not being applied to variant images.
+  * An issue with signing variables not expanding strings for nRF70 Series devices.
+
+* Deprecated the following devicetree properties:
+
+  * ``owner-id``
+  * ``perm-read``
+  * ``perm-write``
+  * ``perm-execute``
+  * ``perm-secure``
+  * ``non-secure-callable``
+
+  It is recommended to replace them with the new devicetree property: ``nordic,access``.
+  See the :ref:`migration guide <migration_2.8_recommended>` for more information.
+
+* Removed the non-working support for configuring the NSIB signing key through the environmental or command line variable (``SB_SIGNING_KEY_FILE``) along with child image.
```

Comment:
This is a bit of an odd thing because nothing has actually changed, the feature did not do anything and has never worked, nothing has been done. I don't remember if a documentation page was removed or not, but this reads like a feature was removed when it wasn't

Reply:
So what changed here? If nothing has changed and the feature did nothing and has never worked? Maybe we don't need this entry then?

Reply:
Discussed offline with Jamie, the related doc was removed. Hence, update the release entry accordingly.